backend/worker/requirements.txt

```diff
@@ -3,7 +3,10 @@ requests==2.24.0
 mitmproxy==7.0.3
 cryptography==3.3.2
 pytest==6.0.1
-# TODO: once scrapy releases version 2.6.0 with this change in it (https://github.com/scrapy/scrapy/issues/5267), switch back to using scrapy from PyPI.
-git+https://github.com/scrapy/scrapy.git@9828cb3f0bcc084c99692f19e65c2ff61a06b35a
+scrapy==2.3.0
```

Comment:
revert scrapy upgrade, that's taken care of in https://github.com/cisagov/crossfeed/pull/1402